### PR TITLE
taxonomy: ro translations

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**
### Part of
-

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -14353,6 +14353,7 @@ nn:ghi, ghee
 pa:ਘਿਉ
 pl:ghi, ghee
 pt:ghi, ghee
+ro:ulei de unt
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:14356]: Do you think that we could add "ghee" as a synonym as well?
```suggestion
ro:ulei de unt, ghee
```

Comment (aleene) [reply taxonomies/ingredients.txt:14356]: I rather keep them separate, just to keep the indian variant as its own entity. You could add it to butter fat.

**Comment by alexgarel** [comment]
@benbenben2 and @aleene would you approve ?

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (benbenben2) [reply taxonomies/ingredients.txt:14356]: @alexgarel, 
I understood that @aleene suggests to keep only ghee in this block (indian variant) and more ulei de unt in other entry (butter fat)

Comment (benbenben2) [reply taxonomies/ingredients.txt:14356]: suggestion from @aleene has been applied (you can ignore the first comment of this thread)

**Comment by benbenben2** [comment]
@alexgarel , @moon-rabbitOFF , @aleene 

As suggested by @aleene, "ulei de unt" was move under "en:butterfat" (which already contains "butter oil" ("ulei de unt" in English) as synonym).

And the other block where "ulei de unt" was located, now contains only "ghee" for all languages.

**Comment by moon-rabbitOFF** [comment]
Thank yu for your changes @benbenben2

**Review by benbenben2 — APPROVED** [review]

**Pull request merged**